### ISSUE TITLE
Window too small on linux 

### ISSUE THREAD

**Issue by Schweiger-Jacob** [quozl/netrek-client-cow#9]
my screensize is 3072 x 1920 and the game is too small to play. I downloaded the gaming using sudo apt install netrek-client-cow. Is there a config file to increase it's size?

**Comment by quozl** [comment]
No.  The graphics does not scale yet.  Try the full screen toggle key, <kbd>"</kbd>.  It changes display resolution to 1024 x 768, and your display adapter or monitor should then rescale.
